### PR TITLE
Translations update from Weblate

### PR DESCRIPTION
**Pull request by translate-h5p**
Translations update from [Weblate](https://translate-h5p.tk/weblate/projects/h5p/h5p-ar-scavenger/) for H5P/h5p-ar-scavenger.

Current translation status:

Weblate translation status

### CODE REVIEW

File: language/es-mx.json [hunk 1]
```diff
@@ -156,7 +156,10 @@
                   {
                     "label": "Rango del Puntaje"
                   },
-                  {},
+                  {
```
Comment (otacke) [inline language/es-mx.json:159]: lines 159-162 are redundant and lead to mismatch with .en.json

File: language/es.json [hunk 2]
```diff
@@ -156,7 +156,10 @@
                   {
                     "label": "Rango del Puntaje"
                   },
-                  {},
+                  {
```
Comment (otacke) [inline language/es.json:159]: lines 159-162 are redundant and lead to mismatch with .en.json

File: language/th.json [hunk 3]
```diff
@@ -0,0 +1,269 @@
+{
```
Comment (otacke) [inline language/th.json:1]: This file has severe mismatches with .en.json.

**Review by otacke — CHANGES_REQUESTED** [review]

**Comment by sr258** [comment]
Sorry about that. Not sure why Weblate didn't detect the structural change in .en.json. It's possibly caused by the empty object...

It's also very strange that the Thai structure was completely wrong. Was there a recent big change to .en.json?

**Comment by otacke** [comment]
@sr258 Nothing to worry about! Yes, I moved some properties higher up in the hierarchy and added some others lately.

**Review by otacke — APPROVED** [review]
Excellent! Thanks!

To avoid issues, it might be better not to use the master branch - at least for those content types that have not yet been officially released. I may not always update all the translation files /the template at once when developing, and actually having translations at that stage makes developing harder as I'll have to take care of them instead of inviting translations when the content type is "done"

**Comment by sr258** [comment]
The problem is that if I use the release branch instead, PRs will also target the release branch.

**Comment by otacke** [comment]
Ah. I see. That's unfortunate indeed.

**Pull request merged**